k8s/prod/common/dg/dg-docassembly.yaml

```diff
@@ -1,40 +0,0 @@
----
-apiVersion: helm.fluxcd.io/v1
-kind: HelmRelease
-metadata:
-  name: dg-docassembly
-  namespace: dg
-  annotations:
-    flux.weave.works/automated: "true"
-    flux.weave.works/tag.java: glob:prod-*
-spec:
-  releaseName: dg-docassembly
-  rollback:
-    enable: true
-    retry: true
-  chart:
-    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
-    name: dg-docassembly
-    version: 1.1.0
-  values:
-    java:
-      replicas: 2
-      memoryRequests: "512Mi"
-      cpuRequests: "250m"
-      memoryLimits: "2048Mi"
-      cpuLimits: "1500m"
```

Comment:
Do we need this cpuLimits? As default value is 1000m. https://github.com/hmcts/chart-java

Reply:
Added to deafult.